spec/javascripts/NewService/formElements/BackendApiSelect.spec.jsx

```diff
@@ -1,40 +1,26 @@
 // @flow
 
 import React from 'react'
-import {act} from 'react-dom/test-utils'
 import Enzyme, {shallow} from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 
 import {BackendApiSelect} from 'NewService/components/FormElements'
-import { BASE_PATH } from 'NewService'
 
 Enzyme.configure({adapter: new Adapter()})
 
-const props = {
-  backendApis: []
-}
+const backendApis = [
+  { id: 1, name: 'backend 1', link: '', type: 'backend' },
+  { id: 2, name: 'backend 2', link: '', type: 'backend' }
+]
 
 it('should render itself', () => {
-  const wrapper = shallow(<BackendApiSelect {...props} />)
-  expect(wrapper.find('#service_act_as_product_input').exists()).toEqual(true)
+  const wrapper = shallow(<BackendApiSelect backendApis={backendApis} />)
+  expect(wrapper.exists()).toBe(true)
 })
 
-it('should render Act as Product checkbox', () => {
-  const wrapper = shallow(<BackendApiSelect {...props} />)
-  expect(wrapper.find('input[name="service[act_as_product]"]').exists()).toEqual(true)
-})
-
-it('should not render select element when Act as Product checkbox is unchecked', () => {
-  const wrapper = shallow(<BackendApiSelect {...props} />)
-  expect(wrapper.find('select[name="service[backend_api]"]').exists()).toEqual(false)
-})
+it('should render select element for backend apis with an empty option', () => {
```

Comment:
1st option is showing text "Create a new Backend API" is not empty

Reply:
_empty option_ refers to one that hasn't any value, which is the case.

Reply:
Then maybe "without value"